layouts/shortcodes/page_gallery.html

```diff
@@ -1,21 +1,8 @@
 {{ $shortcode := . }}
-
-<div class="gallery">
-  {{ $section := .Page.GetPage (.Get "pages") }}
-  {{ range (or $section.Pages ($section.Resources.ByType "page")) }}
-    <div class="entry">
-      {{ $link_internal := eq ($shortcode.Get "link") "internal" }}
-      {{ $link_url := or (and $link_internal .RelPermalink) .Params.repository }}
-      <a href="{{ $link_url }}" class="name">
-        <div class="photo">
-          <img
-            src="{{ default (relURL "/images/avatar.png") (.Params.avatar) }}"
-            loading="lazy"
-            alt="Avatar of {{ .Title }}"
-          />
-        </div>
-        {{ .Title }}
-      </a>
-    </div>
-  {{ end }}
-</div>
+{{ $items := slice }}
+{{ $section := .Page.GetPage (.Get "pages") }}
+{{ range (or $section.Pages ($section.Resources.ByType "page")) }}
+  {{ $body := printf "{{< image >}}\n src = '%s'\n alt = '%s Logo'\n {{< /image >}}" .Params.avatar .Title }}
```

Comment:
Does the image shortcode support `loading="lazy"`?

Reply:
I can check, but we can easily add it.

Reply:
See #587